code/game/objects/items/weapons/tools/screwdrivers.dm

```diff
@@ -1,41 +1,42 @@
 /obj/item/weapon/tool/screwdriver
 	name = "screwdriver"
-	desc = "You can use this to open panels and such things."
+	desc = "You can use this to open panels, screw stuff and such things."
 	icon_state = "screwdriver"
 	flags = CONDUCT
 	worksound = WORKSOUND_SCREW_DRIVING
 	slot_flags = SLOT_BELT | SLOT_EARS
 	w_class = ITEM_SIZE_TINY
-	throw_speed = 3
-	throw_range = 5
 	matter = list(MATERIAL_STEEL = 2, MATERIAL_PLASTIC = 1)
 	attack_verb = list("stabbed")
 	tool_qualities = list(QUALITY_SCREW_DRIVING = 30, QUALITY_BONE_SETTING = 10)
 
 /obj/item/weapon/tool/screwdriver/improvised
 	name = "screwpusher"
-	desc = "A little metal rod wrapped in tape, barely qualifies as a tool."
+	desc = "A little metal rod wrapped in tape, barely qualifies as a tool. This can be fixed with enough tool mods, for which it has ample capacity."
 	icon_state = "impro_screwdriver"
 	tool_qualities = list(QUALITY_SCREW_DRIVING = 15)
 	degradation = 2
+	max_upgrades = 5 //all makeshift tools get more mods to make them actually viable for mid-late game
 
 /obj/item/weapon/tool/screwdriver/electric
 	name = "electric screwdriver"
-	desc = "Screwdriver powered by S class cell."
+	desc = "Electric screwdriver powered by S class cell. Can be used as drilling tool if neccessary, thought not well suited for it."
```

Comment:
"An electrical screwdriver, powered by an S class cell. Can be used as a drilling tool if necessary, though is not well suited to do so."